frontend/app/src/components/settings/data-security/ChangePassword.vue

```diff
@@ -107,9 +100,9 @@ const change = async () => {
         :error-messages="toMessages(v$.newPasswordConfirm)"
         variant="outlined"
       />
-    </VForm>
+    </form>
 
-    <template #buttons>
+    <template #footer>
       <RuiButton
         depressed
```

Comment:
we don't have this prop atm

Reply:
hmm did I changed that after I tested the component?

Reply:
Oh ok ok, I see there is a leftover property from pre-migration